max72xx/max72xx.go

```diff
@@ -0,0 +1,96 @@
+// Driver works for max7219 and 7221
+// Datasheet: https://datasheets.maximintegrated.com/en/ds/MAX7219-MAX7221.pdf
+package max72xx
+
+import (
+	"machine"
+)
+
+type Device interface {
+	Configure()
+	WriteCommand(register, data byte)
+
+	StartShutdownMode()
+	StopShutdownMode()
+	StartDisplayTest()
+	StopDisplayTest()
+	SetDecodeMode(digitNumber uint8)
+	SetScanLimit(digitNumber uint8)
+}
+
+type device struct {
+	bus  machine.SPI
+	load machine.Pin // load
+}
+
+// NewDriver creates a new max7219 connection. The SPI wire must already be configured
+// The SPI frequency must not be higher than 10MHz.
+func NewDevice(load machine.Pin, bus machine.SPI) Device {
```

Comment:
The other SPI drivers generally have the SPI bus param first. Also isn't "load" really the chip select pin? e.g.

```go
func NewDevice(bus machine.SPI, cs machine.Pin) Device
```

Reply:
I can change the order of the parameters. 

The load pin actually is the cs pin. 
The datasheet refers to it as load (cs) pin, which is why i used the same wording. But i can also just rename it to cs

Reply:
fixed :)